packages/app/src/systems/UI/components/NumberInput.tsx

```diff
@@ -31,7 +31,7 @@ export function NumberInput({
         placeholder={placeholder}
         thousandSeparator={thousandSeparator}
         displayType={disabled ? "text" : "input"}
-        onValueChange={(e) => onChange?.(e.value)}
+        onValueChange={(e: any) => onChange?.(e.value)} // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
`(e)` was not typed here?

Reply:
iirc I made this change bc it was giving me an implicit any error

Reply:
weird I'm not running into that error anymore so I'll change it back